src/main/resources/default/taglib/k/section.html.pasta

```diff
@@ -1,15 +1,24 @@
 <i:arg type="String" name="heading" default="" description="May contain a heading of this section."/>
 <i:arg type="String" name="anchor" default="" description="Defines an anchor to which a link can directly jump."/>
+<i:arg type="String" name="borderColor" default="" description="Defines the border color via a declarative css-class."/>
+<i:arg type="String" name="headingColor" default="" description="Defines the text color via a declarative css-class."/>
```

Comment:
Ich weiß ihr habt das wahrscheinlich so abgestimmt aber ich würde die Params in der k:section `class` und `headingClass` nennen und dann in den infoSection und warnSection Templates auch das `card-border` im `class` Parameter mitgeben.

Dann ist es etwas flexibler (von der Benennung her) und entspricht mehr der Funktionsweise anderer Tycho Tags.

Reply:
Habs umgesetzt!